crm_phonecall/views/crm_phonecall_view.xml

```diff
@@ -296,6 +331,12 @@
                         context="{'group_by':'date'}"
                         help="Calls Date by Month"
                     />
+                    <filter
+                        string="State"
+                        name="state"
```

Comment:
```suggestion
                        name="groupby_state"
```

Can you rename the others group by as well, for consistancy?

I think that's what's causing the issue reportred by @skeller1 
There's a context key in the action: `'search_default_state': 'done'`.
The intention is to apply a default state filter, but it's also triggering the group by filter because it has the same name.

Reply:
I have sent a commit fixing this @ivantodorovich